model/Licensing/Classes/CopyrightTextField.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CopyrightTextField
```

Comment:
Interesting approach to covering the `NONE` and `NOASSERTION` values.  I don't have an opinion yet on this approach, but it is a bit different than the SPDX 2.X approach.

I'll open a separate issue in the model to discuss a consistent approach for the `NONE` and `NOASSERTIONS`.